jetstream/tests/test_config.py

```diff
@@ -55,6 +58,19 @@ def test_get_nonexisting_data_source(self):
         with pytest.raises(Exception):
             ConfigLoader.get_data_source("non_existing", "foo") is None
 
+    def test_as_of(self):
```

Comment:
Unless I'm missing something, it doesn't seem necessary to test something that lives entirely in `metric-config-parser` here.